Pipfile

```diff
@@ -24,7 +24,9 @@ atomicwrites = "*"
 validators = "*"
 yamlfix = "*"
 yamllint = "*"
-
+pytablewriter = "*"
+importlib-metadata = "*"
+typed-ast = "*"
```

Comment:
Please add a one line comment explaining why importlib-metadata and typed-ast needs to be added.

Reply:
Added @peterzhuamazon